app/models/manageiq/providers/embedded_terraform/automation_manager/job.rb

```diff
@@ -40,15 +40,35 @@ def poll_runner
   end
 
   def post_execute
-    cleanup_git_repository
-
-    return queue_signal(:finish, message, status) if success?
+    if success?
+      logs_update
+      return queue_signal(:finish, message, status)
+    end
 
     _log.error("Failed to run template: [#{error_message}]")
 
     abort_job("Failed to run template", "error")
   end
 
+  def logs_update
+    terraform_response = {
+      stack_id: stack_response&.response&.stack_id,
+      stack_name: stack_response&.response&.stack_name,
+      status: stack_response&.response&.status,
+      stack_job_end_time: stack_response&.response&.stack_job_end_time,
+      stack_job_start_time: stack_response&.response&.stack_job_start_time,
+      created_at: stack_response&.response&.created_at,
+      message: stack_response&.response&.message
+    }
+
+    miq_task.context_data ||= {}
+    miq_task.context_data[:terraform_respons] = terraform_response
+    context[:terraform_respons] = terraform_response
+    
+    update!(:context => context, :started_on => context[:terraform_respons][:stack_job_start_time])
```

Comment:
Do we want to override the started_on property?  Was this not being set before?  If it wasn't being set we probably want to set it earlier than `post_execute`

Reply:
E.g. embedded_ansible sets this on execute https://github.com/ManageIQ/manageiq/blob/master/app/models/manageiq/providers/ansible_runner_workflow.rb#L50-L52